app/services/test_service/create_content.rb

```diff
@@ -0,0 +1,40 @@
+module TestService
+  class CreateContent < BaseService
+    def initialize(params)
+      super
+      @params = params
+    end
+
+    def call
+      begin
+        test = Test.find(@params[:test_id])
+      rescue ActiveRecord::RecordNotFound
+        add_error('Test not found')
+        return
+      end
+
+      ActiveRecord::Base.transaction do
+
+        test_multiple_choice = TestMultipleChoice.new file_path: @params[:multiple_choice][:file_path], score: @params[:multiple_choice][:score]
+        test_multiple_choice.save!
+        answers = []
+
+        @params[:multiple_choice][:answers].each do |answer|
```

Comment:
What if the params structure is not exactly like what this code expected?

Reply:
yes it's my problem, I'm finding a library can validate params, before pass it to service